taginfo.json

```diff
@@ -2788,6 +2788,14 @@
       ],
       "description": "Allows access for hov."
     },
+    {
+      "key": "payment:*",
+      "object_types": [
+        "way",
```

Comment:
we are ignoring it on the way for now right?

Reply:
oops, right